about.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./Styles/about.css" type="text/css">
+    <script src="https://kit.fontawesome.com/c2d0b29889.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Daniel Yerimah's Capstone Project</title>
+</head>
+<body>
+    <header>
+    <div class="topnav">
+        <ul>
+            <li><a href="#" class="fa fa-facebook"></a></li>
+            <li><a href="#" class="fa fa-twitter"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+    </div>
+    <div class="container">
+    <div class="navbar">
+        <div class="logo">
+           <a href="index.html"><img src="./images/newlogo.png" alt="African Tech Summit Logo"></a>
+        </div>
+        <nav>
+            <ul>
+                <li><a href="about.html">About</a></li>
```

Comment:
- [ ] Kindly change the about into home and add a link to the index.html 

`<li><a href="about.html">About</a></li>` into `<li><a href="index.html">Home</a></li>`

Reply:
changes made